test/lib/api_wrapper_test.rb

```diff
@@ -0,0 +1,145 @@
+require 'test_helper'
+require 'api_wrapper'
+
+class APIWrapperTest < ActionController::TestCase
+
+  test 'the truth' do
+    assert true
+  end
+
+# can't test this since the method can't change constants??
+  # test 'search raises argument error if the incorrect key or ID is passed' do
+  #   VCR.use_cassette('messed-up-url') do
+  #     assert_raises(ArgumentError) do
+  #       APP_ID = "ksresdfjsdfk"
+  #       APIWrapper.search("cat")
+  #     end
+  #   end
+  # end
+
+  test 'make_recipe_list returns a non-empty array if a reasonable term is searched' do
+    # skip
+    VCR.use_cassette('chicken') do
+      recipe_list = APIWrapper.make_recipe_list(APIWrapper.search('chicken'))
+      assert(recipe_list.is_a?(Array))
+      assert_not_empty(recipe_list)
+    end
+  end
+
+  test 'make_recipe_list returns an array of Recipe objects' do
+    # skip
+    VCR.use_cassette('chicken') do
+      recipe_list = APIWrapper.make_recipe_list(APIWrapper.search('chicken'))
+
+      recipe_list.each do |recipe|
+        assert(recipe.is_a?(Recipe))
+      end
+    end
+  end
+
+  test 'search returns the correct length of array (this based on an API call to Postman)' do
```

Comment:
This one you would need to be careful on since if they add more recipes to Edamam the number of responses will change, if you delete the cassette.